packages/ui/src/components/accounts/ImportMnemonic/ImportMnemonic.tsx

```diff
@@ -1,88 +1,77 @@
-import {Button, createStyles, Grid, TextField, Typography} from '@material-ui/core';
-import {makeStyles} from '@material-ui/core/styles';
+import { Button, createStyles, Grid, TextField, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import * as React from 'react';
-import {ConfirmedPasswordInput} from "../../../index";
-import {WithDefaults} from "@emeraldwallet/core";
+import { ConfirmedPasswordInput } from '../../../index';
+
+interface OwnProps {
+  isValidMnemonic?: (mnemonic: string) => boolean;
+  onSubmit: (mnemonic: string, password: string | undefined) => void;
+}
 
 const useStyles = makeStyles(
   createStyles({
     saveButton: {
-      width: "200px",
-      margin: "5px"
+      width: '200px',
+      margin: '5px',
     },
-  })
+  }),
 );
 
-// Component properties
-interface OwnProps {
-  classes?: any;
-  onSubmit: (mnemonic: string, password: string | undefined) => void;
-  isValidMnemonic?: (mnemonic: string) => boolean;
-}
-
-const defaults: Partial<OwnProps> = {
-  classes: {},
-  isValidMnemonic: (mnemonic: string) => false
-}
-
-/**
- *
- */
-const Component = ((props: OwnProps) => {
-  props = WithDefaults(props, defaults);
+const Component: React.FC<OwnProps> = ({ isValidMnemonic, onSubmit }) => {
   const styles = useStyles();
-  const {classes} = props;
 
-  const [mnemonic, setMnemonic] = React.useState("");
-  const [password, setPassword] = React.useState("");
+  const [mnemonic, setMnemonic] = React.useState('');
+  const [password, setPassword] = React.useState('');
   const [done, setDone] = React.useState(false);
 
-  const filteredMnemonic = mnemonic.trim().replace(/\s{2,}/g, ' ');
-  const isValidMnemonic = props.isValidMnemonic(filteredMnemonic);
+  const filteredMnemonic = mnemonic.trim().replace(/\n*\s+/g, ' ');
```

Comment:
doesn't just `/\s+/` work? 
`\s` usually mean any white-space character, and the problem with the previous regex was that it was accepting only 2+ characters, while `\n` is just one